helm/values.yaml

```diff
@@ -0,0 +1,52 @@
+---
+replicaCount: 1
+
+securityContext:
+  fsGroup: 123
```

Comment:
Regarding to the configuration of this number in Dockerfile of core app is not so easy and doesn't expect to change it in future, it can be static in the manifest file.